### ISSUE TITLE
Error while converting .pb model into tensorflowlite model

### ISSUE THREAD

**Issue by Jigar27** [tensorflow/tensorflow#20470]
Command:
 toco --input_file=/home/p/Downloads/tensorflow/output_graph.pb --output_file=/home/p/Downloads/tensorflow/optimized.tflite --input_format=TENSORFLOW_GRAPHDEF --output_format=TFLITE --input_arrays="dense_1_input" --output_arrays="act_4/Softmax" --inference_type=FLOAT --input_data_type=FLOAT --input_shapes=1,40

- output_graph.pb is a frozen model.

Error:
2018-06-29 16:02:00.398756: F tensorflow/contrib/lite/toco/model_cmdline_flags.cc:260] Check failed: uses_single_input_flags 
Aborted (core dumped)

Any help will be highly appreciated.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by shashishekhar** [comment]
@Jigar27: can you provide the graph that you are using.

**Comment by shashishekhar** [comment]
Closing due to lack of activity, @Jigar27 please reopen if you are still having issues.
